lib/std/private/gitutils.nim

```diff
@@ -4,7 +4,8 @@ internal API for now, API subject to change
 
 # xxx move other git utilities here; candidate for stdlib.
 
-import std/[os, osproc, strutils, tempfiles]
+import std/[os, osproc, strutils, random]
+from std/tempfiles import genTempPath
```

Comment:
`from std/tempfiles {.all.} import genTempPath` seems to cause booting issue. So I have to public `genTempPath`.